src/Nethermind/Nethermind.Init/Steps/InitializeNetwork.cs

```diff
@@ -66,13 +66,23 @@ public class InitializeNetwork : IStep
     private readonly ILogger _logger;
     private readonly INetworkConfig _networkConfig;
     protected readonly ISyncConfig _syncConfig;
-
-    public InitializeNetwork(INethermindApi api)
+    private readonly IProtocolsManager _protocolsManager;
+    private readonly IProtocolValidator _protocolValidator;
+    private readonly EthaProtocolFactory _ethaProtocolFactory;
+
+    public InitializeNetwork(
+        INethermindApi api,
+        IProtocolsManager protocolsManager,
+        IProtocolValidator protocolValidator,
+        EthaProtocolFactory ethaProtocolFactory)
```

Comment:
We don't need it, this class shouldn't be changed at all

Reply:
You can simply undo all changes in this class. What to do instead is described in other comments :)